install/kots/manifests/gitpod-shiftfs-module-loader.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+# This job is allowed to fail as will default to fuse
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: shiftfs-module-loader-{{repl Cursor }}
+  labels:
+    app: gitpod
+    component: shiftfs-module-loader
+spec:
+  activeDeadlineSeconds: 30
+  backoffLimit: 1
+  parallelism: 1
+  template:
+    metadata:
+      labels:
+        app: gitpod
+        component: shiftfs-module-loader
+    spec:
+      serviceAccountName: installer
+      restartPolicy: Never
+      containers:
+        - name: shiftfs-module-loader
+          image: eu.gcr.io/gitpod-core-dev/build/shiftfs-module-loader:commit-ab235e8bc00f2c1ac70232cea17e5df9a9d262fa
```

Comment:
This is the current valid tag. This is generated by the Werft job

Reply:
NIT: Would love to see this as a comment in the YAML file as well! :innocent: :stuck_out_tongue_closed_eyes:

Reply:
Nit resolved, hopefully to your satisfaction